src/AppServices/DataSciencePage/DataSciencePage.tsx

```diff
@@ -29,16 +30,28 @@ export const DataSciencePage: FunctionComponent = () => {
         tagLine={t("datascienceoverview:heroTagline")}
         description={t("datascienceoverview:heroDescription")}
         cta={
-          <Button
-            data-testid="hero-buttonLearnMore"
-            variant={ButtonVariant.secondary}
-            component="a"
-            href="https://www.redhat.com/en/technologies/cloud-computing/openshift/openshift-data-science"
-            target="_blank"
-          >
-            {t("datascienceoverview:heroCallToActionButton")}{" "}
-            <ExternalLinkAltIcon className="pf-u-ml-sm" />
-          </Button>
+          <Flex>
+            <Button
+              data-testid="hero-buttonTryIt"
+              ouiaId="button-rhoads-tryit"
```

Comment:
We are trying to have tests that explicitly check for changes of OUIA ids whenever we add one. Would you like trying adding one for this? It should be something along the lines of [this test](https://github.com/redhat-developer/app-services-ui-components/blob/dbb11acf6e58ae5a5e3d70b708db43a5e37d4686/src/Kafka/CreateKafkaInstance/CreateKafkaInstance.test.tsx#L9)